config/maintainers.json

```
@@ -30,6 +30,16 @@
       "link_url": null,
       "avatar_url": null,
       "bio": null
+    },
+    {
+      "github_username": "neenjaw",
+      "alumnus": false,
+      "show_on_website": true,
+      "name": "Tim Austin",
+      "link_text": "https://github.com/neenjaw",
+      "link_url": null,
+      "avatar_url": null,
+      "bio": "Exercism.io has been a great way for me to stay current and expand my developer experience in new areas.  I am excited to be working with the elixir track, because I appreciate the clarity/idiomaticity of the elixir language.  I want others to enjoy this track as much as I have."
```

```suggestion
      "bio": "Exercism has been a great way for me to stay current and expand my developer experience in new areas.  I am excited to be working with the elixir track, because I appreciate the clarity/idiomaticity of the elixir language.  I want others to enjoy this track as much as I have."
```

Changing to "Exercism" as that's the official wording :)

---

Ah - I need you to accept the suggestion :)